lib/facility.rb

```diff
@@ -1,14 +1,84 @@
+require './lib/vehicle'
+require './lib/registrant'
+require 'date'
+
 class Facility
-  attr_reader :name, :address, :phone, :services
+  attr_reader :name, :address, :phone, :collected_fees, :registered_vehicles
+  attr_accessor :services
 
-  def initialize(name, address, phone)
-    @name = name
-    @address = address
-    @phone = phone
+  def initialize(hash, collected_fees = 0)
+    @name = hash[:name]
+    @address = hash[:address]
+    @phone = hash[:phone]
     @services = []
+    @collected_fees = collected_fees
+    @registered_vehicles = []
   end
```

Comment:
I like seeing that you've found out how to set default values upon initialization. Looks good!